config/config.json.in

```diff
@@ -381,6 +381,21 @@
     "Svinval": {
       "supported": true
     },
+    "Ssnpm": {
+      "supported": @CONFIG_XLEN_IS_64@,
+      "supported_pmlen_7": true,
+      "supported_pmlen_16": true
```

Comment:
As was pointed out earlier, these names bake in the `pmlen` values for `xlen==64`.  This will cause complications later when we have to support wider `xlen`s, since downstream users would have started using these config names.   Perhaps we could use self-documenting names like `supported_pmlen_xlen_minus_{48,57}` (which you might have had earlier), or @radimkrcmar's suggestion of specifying a `legal_pmm` array  (I slightly prefer the self-documenting names).